packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -104,7 +104,7 @@ export class DocumentStorageService implements IDocumentStorageService {
     }
 
     public async readBlob(blobId: string): Promise<ArrayBufferLike> {
-        const iso = IsoBuffer.from(await this.read(blobId), "base64");
```

Comment:
This does not look right. THis code works with binary data. It should either stay as is, or can be refactored but has to do same thing as before. With this change, it treats binary date as string, which will not work